### DISCUSSION TITLE
ChatCompletionResponse' object has no attribute 'model_dump_json'

### DISCUSSION THREAD

**Discussion by ecsfu** [zai-org/ChatGLM3#1021] [Q&A] [answered]
yield "{}".format(chunk.model_dump_json(exclude_unset=True))    

The code is throwing an error at this point. What's the issue?

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
检查你的pydantic版本

**Reply by Sailing-hey** [reply]
> 检查你的pydantic版本

pydanitc用什么版本的
